src/item.cpp

```diff
@@ -2082,6 +2083,13 @@ void item::on_wield( player &p, int mv )
         mv += penalty;
     }
 
+    // weapons with a folding stock
+    if( has_flag( "NEEDS_UNFOLD" ) && !is_gunmod() ) {
+        int penalty = std::max( 0, 300 - p.get_skill_level( gun_skill() ) * 10 );
```

Comment:
Just noticed: need to check `is_gun()`, like with `SLOW_WIELD` above.

Will work now without it, but if an item that is not a gun is introduced with `NEEDS_UNFOLD`, results might be unexpected.